edx_rest_framework_extensions/settings.py

```diff
@@ -33,3 +38,25 @@ def get_setting(name):
         return getattr(settings, 'EDX_DRF_EXTENSIONS', {})[name]
     except KeyError:
         return DEFAULT_SETTINGS[name]
+
+
+def get_jwt_issuers():
+    """
+    Retrieves the JWT_ISSUERS list from settings.  If no list is defined, an attempt is made to
+    return the deprecated JWT configuration settings instead.
+    """
+    # If we have a 'JWT_ISSUERS' list defined, return it
+    jwt_issuers = getattr(settings, 'JWT_ISSUERS', None)
+    if jwt_issuers:
+        return jwt_issuers
+
+    # If JWT_ISSUERS is not defined, attempt to return the deprecated settings.
+    logger.info("'JWT_ISSUERS' list not defined, checking for deprecated settings.'")
```

Comment:
Fire a [deprecation warning](https://docs.python.org/2/library/exceptions.html#exceptions.DeprecationWarning) instead of simply logging. We should move to the new format for everything.